exporter/metrics.py

```diff
@@ -2,4 +2,7 @@
 
 cpu_utilization_gauge = Gauge("cpu_utilization", "CPU utilization percentage")
 cpu_frequency_gauge = Gauge("cpu_frequency", "CPU frequency in MHz")
-cpu_temperature_gauge = Gauge("cpu_temperature", "CPU temperature in Celsius")
\ No newline at end of file
+cpu_temperature_gauge = Gauge("cpu_temperature", "CPU temperature in Celsius")
+
+ram_utilization_gauge = Gauge("ram_utilization", "RAM utilization percentage")
+ram_memory_gauge = Gauge("ram_memory", "Total RAM memory installed")
```

Comment:
## Requested Changes
* We need to make the Gauges descriptions more comprehensive. We should include some information on the unit of measure and prefix all of them with the name of our exporter (say `stagent_`). For example:
```suggestion
ram_memory_gauge = Gauge("stagent_ram_memory_bytes", "Total RAM memory installed in bytes")
```